README.md

```diff
@@ -42,7 +42,7 @@ You then need to load the plugin. In `boostrap.php`, something like:
 For a complete setup, add the following to your `App\Controller\AppController`:
 
 ```php
-public $components = ['BootstrapUI.Flash'];
+public $theme = 'BootstrapUI';
```

Comment:
Baking might be better but if we want to stay lazy we could replace the FlashComponent with FlashHelper with code similar to what i had added in core PR.